modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/tree/io/BPlusMetaIO.java

```diff
@@ -85,6 +89,10 @@ private BPlusMetaIO(int ver) {
                 refsOff = REFS_OFFSET;
                 break;
 
+            case 5:
+                refsOff = REFS_OFFSET;
+                break;
+
```

Comment:
Double check: shouldn't `case 5` differ from `case 4`?
Otherwise, why not write 
```
case 4:
case 5:
    refsOff = REFS_OFFSET;
    break;
```
?